storage/replica_command.go

```diff
@@ -1426,13 +1426,18 @@ func (r *Replica) splitTrigger(batch engine.Engine, ms *engine.MVCCStats, split
 		return util.Errorf("unable to write MVCC stats: %s", err)
 	}
 
-	// Copy the last verification timestamp.
-	verifyTS, err := r.GetLastVerificationTimestamp()
+	// Copy the last replica GC and verification timestamps.
+	replicaGCTS, err := r.getLastReplicaGCTimestamp()
+	if err != nil {
+		return util.Errorf("unable to fetch last replica GC timestamp: %s", err)
+	}
+	if err := engine.MVCCPutProto(batch, nil /* &deltaMs */, keys.RangeLastReplicaGCTimestampKey(split.NewDesc.RangeID), roachpb.ZeroTimestamp, nil, &replicaGCTS); err != nil {
```

Comment:
Add a comment that both the GC and verification timestamps are unreplicated, and that that's why there are `nil` stats here.
I also thought we still computed this part somewhere, but maybe we don't? @nvanbenschoten will know.